### PR TITLE
DB interaction

### PR DESCRIPTION
**Pull request by FarahSaeed**
DB interaction and Post form for testing.

### CODE REVIEW

File: app/src/main/java/edu/uga/eu_soccer_explorer/DBEngine.java [hunk 1]
```diff
@@ -13,8 +13,15 @@
 public class DBEngine {
 
     // JDBC driver name
-    static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    //static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```
Comment (subedipiyush) [inline app/src/main/java/edu/uga/eu_soccer_explorer/DBEngine.java]: I guess we should clean all these unnecessary comments.

File: app/src/main/webapp/index.jsp [hunk 2]
```diff
@@ -19,6 +19,13 @@
 
 <body>
 
+      <form  method = "POST">
```
Comment (subedipiyush) [inline app/src/main/webapp/index.jsp]: let's revert this change now that you are done testing your method.

File: app/src/main/resources/config.properties [hunk 3]
```diff
@@ -1,3 +1,3 @@
 db_url=jdbc:mysql://localhost:3306/eu_soccer
-db_username=****
-db_password=****
\ No newline at end of file
+db_username=root
```
Comment (subedipiyush) [inline app/src/main/resources/config.properties]: please revert this as well

File: app/src/main/java/edu/uga/eu_soccer_explorer/MainServlet.java [hunk 4]
```diff
@@ -2,6 +2,7 @@
```
Comment (subedipiyush) [inline app/src/main/java/edu/uga/eu_soccer_explorer/MainServlet.java]: Please revert changes in this file now that you are done testing.

**Comment by FarahSaeed** [comment]
Done.

**Review by subedipiyush — APPROVED** [review]
